src/clic.adoc

```diff
@@ -397,6 +403,37 @@ In this `miselect` offset range:
 4*^| ...
 |===
 
+=== New CSRs
+
+[source]
+----
+       Number  Name         Description
+ (NEW) 0x???   mclaimi      Claim top interrupt
+ (NEW) 0x???   mithreshold  Interrupt enable threshold
+----
+
+==== Claim top interrupt (mclaimi)
+
+Register {mclaimi} has the same value as {mtopi}.
+When this value is not zero,
+reading {mclaimi} has the simultaneous side effect of clearing the pending bit for the reported interrupt identity,
+if possible.
+Writes to {mclaimi} are ignored.
+

```

Comment:
put a todo to check if side effect on read is acceptable, otherwise change to write access

Reply:
There are no read-only accesses to CSRs anyway. The only limitation on side effects is that side effects should not occur if the dest is x0.
As writes are ignored according to the spec, you can use any flavor of CSRRRW/CSRRS/CSRRC or the immediate versions. All will have the same result.